content/webapp/utils/iiif/v3/index.ts

```diff
@@ -631,7 +631,7 @@ export function hasItemType(
 ): boolean {
   return (
     canvases?.some(canvas => {
-      return canvas.painting.some(item => {
+      return canvas?.painting.some(item => {
```

Comment:
np: since you're already inside the `some()` you can be sure the `canvas` exists at this point so the `?` is redundant? Although maybe TS was shouting at you so maybe it isn't. What do I know?